.changeset/khaki-swans-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@channel.io/bezier-react': patch
+---
+
+Remove `overflow:hidden` from `TabItems` in the stylesheet
```

Comment:
This will be included in release changelog, so I suggest describing what bug you fixes so that people know about resolved issue reading changelog.

```suggestion
Fixed a bug where the focus effect of the button in the tab component was cut off.
```

Reply:
Thank you for the suggestion! Fixed it.